### ISSUE TITLE
Gone from chrome / gone from the web store

### ISSUE THREAD

**Issue by fozzibab** [Strav/Change-Colors#28]
Sorry, this is not an issue, but I have no other potential method of contacting the developer. I use this awesome extension daily. I get headaches looking at white backgrounds on my monitor and this thing had been linked to dozens of domains that I visit on a daily/weekly basis. It's now been removed from my chrome installation and it's gone from the web store, so I can't reinstall it through there. Is yhis permanent? Are you going to develop a new version? 

Failing that, can you recommend another extension that will do the same thing? All the others I've found so far just dim the screen, or apply the affect to *every* open tab in a window. This thing actually forces what seems like a theme change to pages. I can't emphasize enough how much this improves my browsing experience. 

Edit: Apparently this *is* an issue!

**Comment by darmach** [comment]
It seem to be an issue after all: the reason it's gone is probably due to chrome upgrade, https://chrome.google.com/webstore/detail/change-colors/jbmkekhehjedonbhoikhhkmlapalklgn

install returns "Invalid manifest"

**Comment by fozzibab** [comment]
Well that's definitely going to be a problem (for me, I mean), if this extension has been abandoned. Oy.

**Comment by myathappy** [comment]
you can download a offline change color, and then install the extension manually. I also very like this extension and have installed this extension on my chrome browse. Through my installed chenge color extension, I get this extenstion's website in chrome webstore. It's website is "https://chrome.google.com/webstore/detail/change-colors/jbmkekhehjedonbhoikhhkmlapalklgn". 
  Besides, the last string of the website, i.e., "jbmkekhehjedonbhoikhhkmlapalklgn" is unique for very extension and you can use this unique string to download this extension from the third part website.
 I  hope my answer can help you.

**Comment by darmach** [comment]
@fozzibab Can you find it anywhere to download offline?

**Comment by myathappy** [comment]
I have give you the website of change color extension at chrome webstore. you also can get the entension at the third party website, for example, https://chrome-extension-downloader.com/ ,with the extension ID.

**Comment by myathappy** [comment]
oh , it is fozzibab that you want to ask. please leave out my answer.

**Comment by darmach** [comment]
@myathappy No, actually, you can't download it off chrome.google.com to use offline, and the https://chrome-extension-downloader.com/ gives me:
```
503 Service Temporarily Unavailable
nginx
```
UPDATE: Got, it - It looks like my DNS was just goofing around. However I still can't install this extension manually because of `Package is Invalid.` - Same reason - wrong manifest.

**Comment by darmach** [comment]
Got, it - It looks like my DNS was just goofing around. Thanks!

**Comment by Fizbin99** [comment]
I fixed the manifest issue and created a CRX file but it won't install anyway. Chrome disables it and it can't be turned on.

**Comment by myathappy** [comment]
@Fizbin99 I have no your problem. My chrome version is  66.0.3359.117.  I think you can try to downgrade your chrome if you want to use this extension.

**Comment by Fizbin99** [comment]
No way I'm rolling back. It's only a matter of time that this will cease to work for everyone.

**Comment by ertay** [comment]
@fizbin99 did you maanage to somehow make it work on Chrome 67? My computer practically became unusable after this extension got disabled for me. I'm visually imapiared and I can't use the browser without this extension. Tried a few others but they are much worse. I wonder if the source on this repository is the latest, perhaps it can be forked and submitted to the Chrome extension store as a new extension?

**Comment by goatcrapp** [comment]
For those having trouble - switch over to vivaldi browser - it's chrome based but  managed separately.  Was able to get change colors installed on it by going to the direct chrome store link provided by myathappy, above.   Perhaps with future updates it will become disabled there as well but for now it works.  It's doubly annoying that google decided to just reach into my browser and remove the extension with no warning or option to backup my settings. I lost settings for hundreds of sites - absolutely BS.

**Comment by Fizbin99** [comment]
@ertay No, I did not. I tried re-writing some of the manifest.json code but it won't work. I suggest using Darkreader. It's not as simple and may take a little more fiddling to get it to work but it's the best alternative. 

https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh

**Comment by ragavanrajan** [comment]
Is there any way to get this extension. I really need this brilliant extension.

**Comment by ertay** [comment]
/u/Gavin_Song on reddit founda a fork of this that works in Chrome 67. You can get it from here:

https://github.com/cuongtructran/Change-Colors

The extension is not published on the Chrome store, so you will need to download the source code from Github and sideload it in Chrome.

In chrome go to chrome://extensions/ then on top right enable 'Developer Mode' and then click Load Unpacaked, and load the folder where the source code is located.

**Comment by Fizbin99** [comment]
@ertay Thanks for the heads up! It does work indeed. I don't think it's quite the same as the old. I notice on some pages like yahoo.com that a lot of the page disappears when I do an override.
